### PR TITLE
Add admin scripts for debugging

### PR DESCRIPTION
**Pull request by GO-Jdang**

### CODE REVIEW

File: Admin/Get-FailedSSRSReports.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*SELECT C.Name, S.LastRunTime, S.LastStatus, S.Description,
+C.Path
+FROM Subscriptions AS S
+LEFT OUTER JOIN [Catalog] AS C
+ON C.ItemID = S.Report_OID
+WHERE LEFT (S.LastStatus, 12) != 'Mail sent to'
+AND LEFT (S.LastStatus, 12) != 'New Subscrip'
+AND LastRunTime > GETDATE() - 1
```
Comment (gnadJ-OG) [inline Admin/Get-FailedSSRSReports.sql]: Is this block of code needed? 
If not, please remove.

File: Admin/Get-FailedSSRSReports.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+@execute_query_database = 'ReportServer',
+@query = N'SELECT C.Name, S.LastRunTime, S.LastStatus, S.Description, C.Path
+FROM Subscriptions AS S
+LEFT OUTER JOIN Catalog AS C
+ON C.ItemID = S.Report_OID
+WHERE LEFT (S.LastStatus, 12) != ''Mail sent to''
+AND LEFT (S.LastStatus, 12) != ''New Subscrip''
+AND LastRunTime > GETDATE() - 1;',
+@attach_query_result_as_file = 1,
+@query_attachment_filename = 'SSRSErrors.txt'
```
Comment (gnadJ-OG) [inline Admin/Get-FailedSSRSReports.sql]: Remove the commented code if it is not needed.

File: Admin/Get-FailedSSRSReports.sql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+@subject = 'T-SQL Query Result',
+@body = 'The result from SELECT is appended below.',
+@execute_query_database = 'ReportServer',
+@query = 'SELECT C.Name, S.LastRunTime, S.LastStatus, S.Description, C.Path
+FROM Subscriptions AS S
+LEFT OUTER JOIN Catalog AS C
+ON C.ItemID = S.Report_OID
+WHERE LEFT (S.LastStatus, 12) != ''Mail sent to''
+AND LEFT (S.LastStatus, 12) != ''New Subscrip''
+AND LastRunTime > GETDATE() - 1;'*/
```
Comment (gnadJ-OG) [inline Admin/Get-FailedSSRSReports.sql]: Remove commented out code if it is not needed.

File: Admin/sp_help_DEMO.sql [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+where login_name = 'tgna\mglitsos'
+GROUP BY login_name, db.name;
+
+SELECT MAX(login_time) AS [Last Login Time], login_name [Login]
+FROM sys.dm_exec_sessions 
+GROUP BY login_name;
+
+
+
+SELECT MAX(login_time) AS [Last Login Time], login_name [Login]
```
Comment (gnadJ-OG) [inline Admin/sp_help_DEMO.sql]: What is the purpose of this select statement?

File: Docs/Lists.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ListItem 1
```
Comment (gnadJ-OG) [inline Docs/Lists.txt]: Please use camelcase and remove the space between ListItem and 1

File: Docs/Lists.txt [hunk 6]
```diff
@@ -0,0 +1,2 @@
...
+ListItem2
```
Comment (gnadJ-OG) [inline Docs/Lists.txt]: Please use camelcase

File: Admin/sp_help_DEMO.sql (hunk 4 above)
Comment (GO-Jdang) [reply Admin/sp_help_DEMO.sql]: This script is to see users logged into the server

File: Docs/Lists.txt (hunk 6 above)
Comment (GO-Jdang) [reply Docs/Lists.txt]: Edited the list item to use camelcase 9756880

File: Docs/Lists.txt (hunk 5 above)
Comment (GO-Jdang) [reply Docs/Lists.txt]: Edited the list item to use camelcase 9756880

**Pull request merged**